### PR TITLE
CORTX: 31879_TE_2

### PR DESCRIPTION
**Pull request by ankita-93**
…rsioning_common_test_lib

Signed-off-by: ankita-93 <ankita.thummar@seagate.com>

# Problem Statement
- TE and run fix https://jts.seagate.com/browse/CORTX-31879 and logs updated https://jts.seagate.com/browse/TEST-43314

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/s3/versioning/test_get_put_object_tagging.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def test_get_put_obj_tags_40429(self):
                                              object_name=self.object_name,
                                              version_tag=self.ver_tag, versions_dict=self.versions)
         assert_utils.assert_true(resp[0], resp)
-
+        put_tag = self.ver_tag[self.object_name][latest_v][-1]
```
Comment (rahul-hatwar) [inline tests/s3/versioning/test_get_put_object_tagging.py:156]: Is this required?

Comment (ankita-93) [reply tests/s3/versioning/test_get_put_object_tagging.py:156]: if we are puting new tag key-val, we supposed to fetch the same before comparing it with GET right ?

**Review by sonalk0209 — APPROVED** [review]

File: tests/s3/versioning/test_get_put_object_tagging.py (hunk 1 above)
Comment (rahul-hatwar) [reply tests/s3/versioning/test_get_put_object_tagging.py:156]: Did you try this test after removing this put_tag?

Comment (ankita-93) [reply tests/s3/versioning/test_get_put_object_tagging.py:156]: This has bin tested, when this new fetch was not there test was failing, as in step 6 above this line we are again doing put tag. Hence we need to fetch the new one before comparing it with new get tag

**Review by rahul-hatwar — APPROVED** [review]

**Pull request merged**